docs/technical_reference/property_models/ASM2D_GHG.rst

```diff
@@ -0,0 +1,321 @@
+.. _ASM2d_GHG:
+
+ASM2D-GHG Property Package
```

Comment:
I noticed that "stripping related parameters" are not yet included. 

For diffusivities, we can use the param or var name `diffus_phase_comp` for consistency. 

For Henry's constants, revisit naming convention we/IDAES uses.

The same goes for partial pressures. 

I am wondering if you omitted these because they correspond to gas phase and only wanted to handle liquid phase at this point.